en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+.. Optionally, to change the input fields, click *Add Input Fields* to open the *Visual Editor* to edit the input fields.
+
+* To define input fields using the Sample *JSON Editor*:
+
+.. In the *Example Request* window, paste or enter the JSON sample. 
++
+.Example: JSON sample in the *Sample JSON Editor* window
+[%collapsible]
+====
+[source,JSON]
+----
+{ 
+{    
+"name": "ben",
+"age": 100
+}
+}
+----
+====
++
+Unlike the *Visual Editor*, the *Sample JSON Editor* supports objects, arrays, and null values. 
++
+.Example: Complex JSON sample in the *Sample JSON Editor* window
+[%collapsible]
+====
+[source,JSON]
+----
+{
+"data": {
+"favoriteNumbers": [3, 7, 13, 17, 23]
+}
+}
+----
+====
++
+.. Click *Apply*. +
+The number of input fields you added appears next to *Input Fields*, displaying one or more new input fields added to the flow.
+.. Optionally, to change the input fields, click *Add Input Fields* to open the *Sample JSON Editor* to edit the input fields.
++
+. Complete the flow. For example, configure system actions for the flow, define variables, or create flow controls. For more information, refer to xref:ms_composer_flows.adoc[Build a Composer Flow]. 
+. Click *Save* to save your flow.
+
+If you need to delete an invocable step from your flow: 
+
+. Access the flow containing the invocable flow step that you want to delete. 
+. In the left corner of the step, click *…* > *Delete step*. +
+The step is deleted.
+ 
+[[test-your-invocable-composer-flow]]
+=== Test Your Invocable Composer Flow 
+
+To test your invocable Composer flow: 
+
+. In your invocable flow, click *Test*. +
+If you have added input fields, the *Test Flow* window appears.
+. Enter a sample value for the input field or review the complex JSON to test the flow.
+. Click *Test Flow*. +
+The testing output appears. Using this output, you can verify if the flow returns the expected values in the sample output of the main canvas. 
+
+
+[[activate-your-invocable-composer-flow]]
+=== Activate Your Invocable Composer Flow 
+
+To activate your invocable Composer flow: 
+
+. In your invocable flow, click *Activate*. +
+The flow details page appears with a message indicating that the flow is now active. 
+. Click *Flow Credentials* to view the flow data that is used to invoke the flow from the external system.
+. Optionally, you can reset the flow credentials periodically to ensure continuous security. In the *Flow Credentials* window, click *Reset Client Secret*. +
+The *Client Secret* field is updated. You can click the clipboard icon next to the field to copy the new *Client Secret* value. 
++
+Once the client secret is reset, the previous client secret no longer works. You must manually share the new client secret to external systems that invoke this Composer flow. 
+. Ensure that the following fields have values: 
+* *URL*: The unique URL of the flow you are invoking. 
```

Comment:
```suggestion
* *URL*: Unique URL of the flow you are invoking. 
```

Reply:
Please remove the articles from the definitions - not necessary to have them

Reply:
Did manually!